dashboard/app/controllers/plc/learning_modules_controller.rb

```diff
@@ -8,13 +8,22 @@ def index
   # GET /plc/learning_modules/1
   # GET /plc/learning_modules/1.json
   def show
-    @resource_tasks, @other_tasks = @learning_module.plc_tasks.partition {|task| task.is_a? Plc::LearningResourceTask}
+    resource_tasks, other_tasks = @learning_module.plc_tasks.partition {|task| task.is_a? Plc::LearningResourceTask}
+    resource_tasks_grouping = {tasks: resource_tasks, name: 'Learning Resources', new_link_text: 'Add a resource',
+                               new_link_destination: plc_learning_module_path(@learning_module) + '/new_learning_resource_for_module'}
```

Comment:
So when I look at localhost/rails/info/routes, I don't see a path helper generated for that route. Is there something I have to do to make it generate?